services/QuillCMS/app/models/response.rb

```diff
@@ -58,6 +58,30 @@ def as_indexed_json(options={})
     }
   end
 
+  def serialized_for_admin_cms(options={})
```

Comment:
I presume that this method signature was copied from elsewhere, but I note that `options` doesn't actually seem to get used in this method.